profiles/dini.js

```diff
@@ -17,6 +17,7 @@ const modules = [
   import("../src/core/markdown.js"),
   import("../src/core/reindent.js"),
   import("../src/core/title.js"),
+  import("../src/core/level.js"),
```

Comment:
```suggestion
```

Reply:
Levels are a W3C thing... not sure DINI will want them too, but @sebastian-meyer can confirm? 

@sebastian-meyer, just FYI, W3C specs sometimes use a (not well defined at all) concept of API/technology "layering"... it's kinda like versioning... but not really. A "level 2 spec" will usually contain of "level 1"'s text, for example:
https://w3c.github.io/input-events/

 Levels are always backwards compatible. This is different to "versions", which can be incompatible with each other.

Reply:
We don't need levels in the DINI profile. DINI is using a rather classical versioning schema.

Reply:
thanks for clarifying, @sebastian-meyer!